components/admin_console/admin_sidebar_header/admin_sidebar_header.jsx

```diff
@@ -28,6 +28,7 @@ export default class SidebarHeader extends React.Component {
         if (me.last_picture_update) {
             profilePicture = (
                 <img
+                    alt={'user profile image'}
```

Comment:
I remember multiple arguments about alt text like https://community-daily.mattermost.com/core/pl/cfwsfcnatt8obmbrhoyihjmjbh and https://community-daily.mattermost.com/core/pl/o83oryu9t3bjup3dcgy99hqiiy

Does this add much value when we have profile name next to it and one more thing is should it say something like `Sudheer's profile image` rather than `user profile image`?

Reply:
@sudheerDev DM'd you.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Attributes and another reliable source https://davidwalsh.name/accessibility-tip-empty-alt-attributes

Excerpt 
> Omitting this attribute altogether indicates that the image is a key part of the content, and no textual equivalent is available. Setting this attribute to an empty string (alt="") indicates that this image is not a key part of the content (decorative), and that non-visual browsers may omit it from rendering.

What i was saying is that we can remove 'user profile image' text and have empty `alt=""`  when next to user name like in this case. This way i feel we make it easier for users who use screen reader as it would skip this.

If we are adding these tags for accessibility reasons we can have empty tags if they are just for decorative purposes.

Reply:
That makes a lot of sense. It's a lot easier to maintain that way as well since we aren't needing to add a lot of extra strings that have to be managed and translated.

Reply:
Don't think the alt attributes need to be translated as such, however, I don't think they're always for decorative purposes, but for this, I think we can probably have the empty string.

Reply:
Why wouldn't we translate them? We translate all text in the UI

Reply:
@hmhealey If we are, then sure, we can. Wasn't sure if we were putting the effort of translating accessibility text as well. But we can then do that as a whole. (I don't think we do that in any place as of yet, but I may be wrong).

Reply:
I guess this is fine. The web app probably isn't usable with a screen reader anyway since I don't think most web apps are, so I don't think anyone will be too affected by this. This is already an improvement over what we had before, and we can always improve it more in the future if we need to.